nasm/README.md

```diff
@@ -0,0 +1,43 @@
+# Netwide Assembler
+
+This is the (Nasm) Netwide assembler and it is meant to be used in the binary compatibility mode of Unikraft.
+
+## Content
+
+- `sample_code.s`: a sample assembly code file that can be assembled using Nasm
+- `usr/bin/nasm`: the Nasm binary
+- `lib/` & `lib64/`: the required libraries for Nasm
+
+## Extract Dynamic Libraries
+
```

Comment:
Add wording. Say: "Use the `extract.sh` script to extract the required dynamic libraries for the binary:"